framework/src/st_things/things_stack/framework/things_data_manager.c

```diff
@@ -1931,7 +1938,7 @@ int dm_load_legacy_cloud_data(es_cloud_signup_s **cl_data)
 const char *dm_get_things_device_type(int device_id)
```

Comment:
We need to change the device_id parameter to "char *".

Reply:
This function is modify in PR #2544